ksql-engine/src/test/java/io/confluent/ksql/planner/plan/ProjectNodeTest.java

```diff
@@ -29,99 +36,120 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import org.apache.kafka.connect.data.Field;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.apache.kafka.streams.StreamsBuilder;
-import org.easymock.EasyMock;
-import org.easymock.EasyMockRunner;
-import org.easymock.Mock;
-import org.easymock.MockType;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
 
-@RunWith(EasyMockRunner.class)
 public class ProjectNodeTest {
 
   @Mock
   private PlanNode source;
-  @Mock(MockType.NICE)
-  private SchemaKStream<String> stream;
+  @Mock
+  private SchemaKStream stream;
 
   private final StreamsBuilder builder = new StreamsBuilder();
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final ServiceContext serviceContext = TestServiceContext.create();
   private final InternalFunctionRegistry functionRegistry = new InternalFunctionRegistry();
   private final HashMap<String, Object> props = new HashMap<>();
+  private final QueryId queryId = new QueryId("project-test");
+
+  @Rule
+  public final MockitoRule mockitoRule = MockitoJUnit.rule();
 
   @Before
   public void init() {
-    EasyMock.expect(source.getNodeOutputType()).andReturn(DataSourceType.KSTREAM);
+    when(source.getNodeOutputType()).thenReturn(DataSourceType.KSTREAM);
   }
 
   @After
   public void tearDown() {
     serviceContext.close();
   }
 
-  @Test(expected = KsqlException.class)
-  public void shouldThrowKsqlExcptionIfSchemaSizeDoesntMatchProjection() {
-    mockSourceNode();
-
-    EasyMock.replay(source, stream);
-
-    final ProjectNode node = new ProjectNode(new PlanNodeId("1"),
+  private ProjectNode buildNode(final List<Expression> expressionList) {
+    return new ProjectNode(
+        new PlanNodeId("1"),
         source,
         SchemaBuilder.struct()
             .field("field1", Schema.OPTIONAL_STRING_SCHEMA)
             .field("field2", Schema.OPTIONAL_STRING_SCHEMA)
             .build(),
-        Collections.singletonList(new BooleanLiteral("true")));
+        expressionList);
+  }
+
+  @Test(expected = KsqlException.class)
+  public void shouldThrowKsqlExcptionIfSchemaSizeDoesntMatchProjection() {
+    mockSourceNode();
```

Comment:
Not your code originally so this is definitely a nit, but perhaps move `mockSourceNode()` into `init()`, since it looks like multiple tests use it and none don't?